### PR TITLE
adds trolcommander (active fork of inactive mucommander)

### PR DESCRIPTION
**Pull request by hovancik**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
This can’t be accepted like this. `home` and `url` are from different repos and they seem to bear no relation to one another. Please explain the cask.

**Comment by hovancik** [comment]
sorry, typo, will fix now

File: Casks/trolcommander.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'trolcommander' do
+  version '0.9.5'
+  sha256 '9bd9fe93245e55d20085ed14d1fafbfa0303806789416a594792e8a206a11457'
+
+  url "https://github.com/trol73/trol-commander-dist/blob/master/#{version}/trolcommander-#{version.gsub('.', '_')}.app.tar.gz?raw=true"
```
Comment (vitorgalvao) [inline Casks/trolcommander.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by hovancik** [comment]
Uh, this got complicated. Hopefully that's all.

**Comment by vitorgalvao** [comment]
Yep. Everything looks fine. Thank you.

**Pull request merged**